docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -156,11 +156,9 @@ Data Asset metrics provide you with insight into the data you can use for your d
 
 2. Click the **Overview** tab.
 
-3. Select one of the following options: 
-
-    - If you have not previously generated Data Asset metrics, click **Fetch Metrics**. 
+    When you select a new Data Asset, abbreviated Data Asset metrics are automatically generated.
```

Comment:
Since the overview page is automatically selected the first time a Data Asset is opened, should this line go above in step 1? Recommend "When you select a new Data Asset, the schema is automatically fetched."

Reply:
@annabarr, the text you identified is referred to as a step result. Step results appear _after_ the action that caused them. Users will see the Data Asset metrics being generated, and the statement will have context and meaning. If the information is presented in the introductory statement, it lacks context and could cause confusion.

Reply:
Works for me. Thoughts on language? We could be more specific and say the schema is fetched automatically, and not metrics

Reply:
@annabarr, if we're changing the terminology from _metrics_ to _schema_, it would probably be best to have a wider conversation. We've been using _metrics_  until this change. Do you want to initiate a conversation on the Slack Product channel? I'll update the docs based on the results of that conversation.

Reply:
We used to call both the schema fetch and the metrics fetch "metrics" because we queried for them at the same time. They are distinct operations, and this work was to fetch the schema automatically and fetch the metrics on demand (because they are more expensive to compute). We can continue to call the schema fetch a subset of "metrics", but it's not exactly right since we are not calculating any numbers.

Reply:
@annabarr will users understand the distinction between metrics and schema? Will they understand they are distinct operations? Will users understand that they are fetching the schema when selecting Profile Data? If the answer to these questions is "no," I recommend keeping the current terminology. Your call.

Reply:
Maybe not. What about "When you select a new Data Asset, columns and data types are automatically fetched."

Reply:
Haha ok sounds good to me!